lib/modals/actions_dialog.dart

```diff
@@ -72,7 +72,7 @@ Future<void> showSelectWithInputDialog(
               child: Padding(
                 padding: const EdgeInsets.all(16),
                 child: SizedBox(
-                  height: 200,
+                  height: 270,
```

Comment:
can this be done relatively to screen size, instead of fixed value?

Reply:
it is centered on the screen so unless someone uses a device that is smaller than 270 we are safe